libnmstate/schemas/operational-state.yaml

```diff
@@ -163,6 +163,13 @@ definitions:
           minimum: 0
         flow-control:
           type: boolean
+        sr-iov:
+          type: object
+          properties:
+            total-vfs:
+              type: integer
+              minimum: 0
+              maximum: 256
```

Comment:
Should this allow 256? I think it shouldn't.

Reply:
The number came from you: https://github.com/nmstate/nmstate/pull/592#discussion_r349574736
Now that you mention it again, I remember that I asked you about it as well:
https://github.com/nmstate/nmstate/pull/592#discussion_r349795251